protocol/common/feature_flags.go

```diff
@@ -17,4 +17,13 @@ type FeatureFlags struct {
 
 	// DisableAutoMessageLoop disables auto message loop
 	DisableAutoMessageLoop bool
+
+	// ResendRawMessagesDisabled indicates whether we should be disabling sending raw messages
+	ResendRawMessagesDisabled bool
+
+	// StoreNodesDisabled indicates whether we should fetch messages from store nodes
+	StoreNodesDisabled bool
+
+	// DisableAdvertisingMessages indicates whether we should advertise and sync messages with other peers
+	DisableAdvertisingMessages bool
```

Comment:
```suggestion
	Peersyncing bool
```

Disable means as if the feature was there before.